UnleashClient/api/features.py

```diff
@@ -15,10 +15,10 @@ def get_feature_toggles(
     instance_id: str,
     custom_headers: dict,
     custom_options: dict,
+    request_timeout: int,
+    request_retries: int,
```

Comment:
Out of curiosity: why change the order of params?

Reply:
the 2 new params don't have any default values so I added them just after all the other params that don't have default value. The 2 params after it have default values. Since this method is not a parto of the public API for the users I felt it's safe to do.

Reply:
Oh nice! I was not paying attention to that detail :+1: